Velocity does not ping correctly

Hello, i run a velocity 3.1 server.

When i try to use the example in the README, i get:

panic: ping error: response error: expected packet 0x0 but got 0x1 instead

--- a/ping/ping.go
+++ b/ping/ping.go
@@ -40,9 +40,15 @@ func sendServerListPing(conn net.Conn, host string, port uint16) (*Response, err
 	if err := writeRequest(conn); err != nil {
 		return nil, fmt.Errorf("request error: %w", err)
 	}
-	if err := writePing(conn, ping{Payload: long(time.Now().Unix())}); err != nil {
-		return nil, fmt.Errorf("ping error: %w", err)
-	}
+
+	//if err := writePing(conn, ping{Payload: long(time.Now().Unix())}); err != nil {
+	//	return nil, fmt.Errorf("ping error: %w", err)
+	//}
+
+	//_, err := readPong(conn)
+	//if err != nil {
+	//	return nil, fmt.Errorf("pong error: %w", err)
+	//}
 
 	res, err := readResponse(conn)
 	if err != nil {

